### DISCUSSION TITLE
math block does not render correctly.

### DISCUSSION THREAD

**Discussion by longyn** [requarks/wiki#6744] [Error / Bug Report]
for example

```markdown
$$
\overrightarrow{p} = (B_x-A_x)\widehat{i}+(B_y-A_y)\widehat{j}+(B_z-A_z)\widehat{k}
$$
```

In editor & preview, render correctly, but in reading view, not render correctly.

**Comment by fnafox** [comment]
Math rendering is probably disabled, check and enable in `Admin > Rendering > Pipeline > Markdown > Mathjax` as needed.
iirc there's currently a bug where the preview renderer displays some disabled options.

**Comment by longyn** [comment]
> Math rendering is probably disabled, check and enable in `Admin > Rendering > Pipeline > Markdown > Mathjax` as needed. iirc there's currently a bug where the preview renderer displays some disabled options.

The `Mathjax` already enabled, still have preview and display issue.

**Comment by BPowell76** [comment]
What I've experienced is Katex works while Mathjax doesn't when it comes to rendering. Both don't work if Latex expressions are part of markdown tables if the expression ends with a `}` which can be a problem if trying to render $e^{x-2}$
